ReadMe.md

```diff
@@ -18,7 +18,7 @@ This plugin is born of my frustaration with the scenario outline in [parcel issu
 
 No additional configuration required. If your app is being bundled by parcel and you have this plugin installed, the processed application will build with:
 
-- the `index.html` containing a `script` tag in its head pointing to the "goodie bag` script (the two polyfills)
+- the `*.html` containing a `script` tag in its head pointing to the "goodie bag` script (the two polyfills)
```

Comment:
Thanks for updating the readme, I tend to think of that after committing the code.